spec/support/capybara.rb

```diff
@@ -3,8 +3,7 @@
 require 'capybara'
 require 'capybara/rspec'
 require 'capybara-screenshot/rspec'
-require 'selenium/webdriver'
-require 'webdrivers'
+# require 'selenium/webdriver'
```

Comment:
question: Is there a reason we need to keep this as a comment? Just wondering for my own curiosity  :)